src/set_theory/game/impartial.lean

```diff
@@ -133,59 +140,54 @@ lemma not_first_loses (G : pgame) [G.impartial] : ¬G.first_loses ↔ G.first_wi
 iff.symm $ iff_not_comm.1 $ iff.symm $ not_first_wins G
 
 lemma add_self (G : pgame) [G.impartial] : (G + G).first_loses :=
-  first_loses_is_zero.2 $ equiv_trans (add_congr (neg_equiv_self G) G.equiv_refl)
-  (add_left_neg_equiv G)
+first_loses_is_zero.2 $ trans (add_congr (neg_equiv_self G) (refl G)) (add_left_neg_equiv G)
 
 lemma equiv_iff_sum_first_loses (G H : pgame) [G.impartial] [H.impartial] :
   G ≈ H ↔ (G + H).first_loses :=
 begin
   split,
   { intro heq,
-    exact first_loses_of_equiv (add_congr (equiv_refl _) heq) (add_self G) },
+    exact first_loses_of_equiv (add_congr (refl _) heq) (add_self G) },
   { intro hGHp,
     split,
     { rw le_iff_sub_nonneg,
       exact le_trans hGHp.2
-        (le_trans add_comm_le $ le_of_le_of_equiv (pgame.le_refl _) $ add_congr (equiv_refl _)
+        (le_trans add_comm_le $ le_of_le_of_equiv le_rfl $ add_congr (refl _)
         (neg_equiv_self G)) },
     { rw le_iff_sub_nonneg,
       exact le_trans hGHp.2
-        (le_of_le_of_equiv (pgame.le_refl _) $ add_congr (equiv_refl _) (neg_equiv_self H)) } }
+        (le_of_le_of_equiv le_rfl $ add_congr (refl _) (neg_equiv_self H)) } }
 end
 
 lemma le_zero_iff {G : pgame} [G.impartial] : G ≤ 0 ↔ 0 ≤ G :=
-by rw [le_zero_iff_zero_le_neg, le_congr (equiv_refl 0) (neg_equiv_self G)]
+by rw [←zero_le_neg_iff, le_congr (refl 0) (neg_equiv_self G)]
+
+lemma lf_zero_iff {G : pgame} [G.impartial] : G ⧏ 0 ↔ 0 ⧏ G :=
+by rw [←zero_lf_neg_iff, lf_congr (refl 0) (neg_equiv_self G)]
 
 lemma lt_zero_iff {G : pgame} [G.impartial] : G < 0 ↔ 0 < G :=
-by rw [lt_iff_neg_gt, pgame.neg_zero, lt_congr (equiv_refl 0) (neg_equiv_self G)]
+by rw [←zero_lt_neg_iff, lt_congr (refl 0) (neg_equiv_self G)]
```

Comment:
Shouldn't this lemma be removed? It says `false <-> false` by `nonneg` and `nonpos`

Reply:
I plan to remove `winner.lean` and refactor `nim.lean` in terms of inequalities instead of winning players in the near future. This PR is already huge as is, so I'm more interested in that it just works for now.